.prettierignore

```diff
@@ -15,6 +15,11 @@ public
 *.json
 *.md
 
+# selectively include instead of all .json because
+# - many (such as jsconfig.json) allow trailing commas want to preserve but Prettier will remove per JSON
+# - some have careful manual formatting such as a metrix and there's no safe way to have an ignore comment in JSON
```

Comment:
What’s Metrix?

Reply:
"matrix" as in https://github.com/Agoric/agoric-sdk/blob/master/packages/swingset-runner/demo/zoeTests/swingset.json

I'll fix before merge

Reply:
Oh, very good. Matrixes vs. matrices: fight! (I favor matrixes since @romancortes and @aadsm convinced me that, when two words are equivalent and accepted, favoring the regular form is easier on an international audience.)